pipelines/templates/build-golang-template.yaml

```diff
@@ -0,0 +1,54 @@
+# This template builds a golang container and pushes it to a container registry, using presidio Makefile.
+# It is used in CI to build the base container as well as all golang based services.
+# parameters:
+#   registry_service_connection: service connection to docker registry.
```

Comment:
is this param relevant when not using Azure?

Reply:
the pipeline is couples to azure and this parameter is the service connection to the publish azure container registry. without this the pipeline can't run